draft-ietf-quic-multipath.md

```diff
@@ -885,6 +884,33 @@ two or more active paths during the connection lifetime. Different applications
 Once the implementation has decided which paths to keep alive, it can do so by sending Ping frames
 on each of these paths before the idle timeout expires.
 
+## Migration
+
+Even when the multipath extension is used, migration can happen.
+E.g. the client's source port and/or IP address may change due to changes
+in network topology or address mappings, such as caused by NAT rebinding.
+In this case the server performs path validation (see {{Section 9 of QUIC-TRANSPORT}}).
+
+Moreover, {{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
+can change the connection ID it uses for a peer to another available one
+at any time during the connection. As such a sole change of the Connection
+ID without any change in the address does indiacte a path. 
+
+If an endpoint uses a new Connection ID after an idle period
+and a NAT binding leads to a 4-tuple changes on the same packet,
+the receiving endpoint may not be able to associate the packet to
+an existing path and will therefore consider this as a new path.
+This leads to an inconsistent view of open paths at both peers,
+however, as the "old" path will not work anymore, it will be silently
+closed after the idle timeout expires. 
+
+{{Section 9.3 of QUIC-TRANSPORT}} allows an endpoint to skip validation of
+a peer address if that address has been seen recently. However, when the
+multipath extension is used and an endpoint has multiple addresses that
+could lead to switching between different paths, it should rather maintain
+multiple open paths instead.
+
+
```

Comment:
I wonder if the following would make it more clear by avoiding the notion of "migration".

Note that I think we could have a better behaviour in the case "both 4-tuple and CID change", but I would prefer delegating this to a later issue.
```suggestion
## NAT Rebinding {nat-rebinding}

While endpoints assign a Connection ID to a specific sending 4-tuple,
networks events such as NAT rebinding may make the packet's receiver
perceives a different 4-tuple. Servers observing 4-tuple changes must
initiate path validation. If the path validation process succeeds, the
endpoint follows considerations of {{Section 9.4 of QUIC-TRANSPORT}}.

## CID Renewal

At any time, endpoints may change the Connection ID they use to send
packets on a path. Such a path can keep the same congestion control
state and RTT measurements.

If an endpoint uses a new Connection ID after an idle period
and a NAT rebinding leads to a 4-tuple changes on the same packet,
the receiving endpoint may not be able to associate the packet to
an existing path and will therefore consider this as a new path.
This leads to an inconsistent view of open paths at both peers,
however, as the "old" path will not work anymore, it will be silently
closed after the idle timeout expires.

```

Reply:
Sorry I did another change and lost you proposal but will edit the PR directly now accordingly. Thx!